types/index.d.ts

```diff
@@ -0,0 +1,61 @@
+import {
+  Color,
```

Comment:
A bit of a nitpick, but would you mind doing 4-space indents in this PR to match the rest of the files?

Reply:
Yes, done. Please check

Reply:
Thanks!